addons/repair/stringtable.xml

```diff
@@ -1884,6 +1883,14 @@
             <Chinesesimp>需要特定物品来移除/更换车轮</Chinesesimp>
             <Chinese>需要特定物品來移除/更換車輪</Chinese>
         </Key>
+        <Key ID="STR_ACE_Repair_patchWheelRequiredItems_DisplayName">
+            <English>Wheel Patch Requirements</English>
+            <German>Bedingungen für die Reifenflicken</German>
```

Comment:
Above its "Radflicken", here its now "Reifenflicken", and further below its "Radflicken" again